app.py

```diff
@@ -44,6 +44,7 @@
 _LOGGER = logging.getLogger("thoth.package_releases")
 _THOTH_METRICS_PUSHGATEWAY_URL = os.getenv("PROMETHEUS_PUSHGATEWAY_URL")
 _THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
+_PROMETHEUS_REGISTRY = os.environ["_PROMETHEUS_REGISTRY"]
```

Comment:
@fridex Can we get `_PROMETHEUS_REGISTRY` as env var?

Reply:
no, but we have: https://github.com/thoth-station/package-releases-job/blob/master/openshift/cronJob-template.yaml#L85